pages/resources/ai.mdx

```diff
@@ -0,0 +1,74 @@
+# AI
+
+AI endpoints for translating messages, paraphrasing thread titles, fixing grammar, and summarizing threads.
+
+## Endpoints
+
+<RouteHeader method="POST" url="/ai/translate">
+  Translate Message
+</RouteHeader>
+
+Translates a message. Returns the translated message.
```

Comment:
```suggestion
Translates a message.
```
"Returns " should only be used when it actually returns a named object. response body is defined, so this is incorrect

applies to rest of eps as well